apps/test/unit/signUpFlow/FinishStudentAccountTest.tsx

```diff
@@ -96,6 +97,13 @@ describe('FinishStudentAccount', () => {
     screen.getByText(locale.go_to_my_account());
   });
 
+  it('does not render state question if user is not detected in the U.S.', () => {
```

Comment:
is it worth adding a test that US state isn't required for non-US IPs?

Reply:
I tried to cover that by also checking that the error message doesn't show up. But if this doesn't feel sufficient I can definitely add tests to this and other questions to make sure the user can click the button without selecting a state (or whichever field the test is about)!

Reply:
Does the error message show up if the user is not in the US and hasn't done any other action?

Reply:
Said another way, if you changed line 101 to be `renderDefault(true)` and removed line 103, would this test fail?

Reply:
Oh I see what you're saying gotcha. If the user is not in the U.S. the error message should never show up, and in the case of doing your suggestion the test would succeed if the user takes no action. I'll add some tests!

Reply:
Adding that sort of test will also prompt the addition of tests to both this file and the teacher side of things so I'm going to make a small ticket as a followup PR!